rapids/test_rapids.py

```diff
@@ -13,25 +14,35 @@ class RapidsTestCase(DataprocTestCase):
 
   GPU_P100 = "type=nvidia-tesla-p100"
 
-  DASK_TEST_SCRIPT_FILE_NAME = "verify_rapids_dask.py"
+  # Tests for RAPIDS init action
+  DASK_RAPIDS_TEST_SCRIPT_FILE_NAME = "verify_rapids_dask.py"
   SPARK_TEST_SCRIPT_FILE_NAME = "verify_rapids_spark.py"
   XGBOOST_SPARK_TEST_SCRIPT_FILE_NAME = "verify_xgboost_spark.py"
 
-  def verify_dask_instance(self, name):
-    self.upload_test_file(
-        os.path.join(
-            os.path.dirname(os.path.abspath(__file__)),
-            self.DASK_TEST_SCRIPT_FILE_NAME), name)
-    self._run_dask_test_script(name)
-    self.remove_test_script(self.DASK_TEST_SCRIPT_FILE_NAME, name)
+  # Tests for validating Dask installation integrity,  under dask/ directory
+  DASK_YARN_TEST_SCRIPT_FILE_NAME = "verify_dask_yarn.py"
+  DASK_STANDALONE_TEST_SCRIPT_FILE_NAME = "verify_dask_standalone.py"
 
-  def _run_dask_test_script(self, name):
-    conda = "miniconda3"
-    if self.getImageVersion() < pkg_resources.parse_version("2.0"):
-      conda = "anaconda"
-    verify_cmd = "/opt/conda/{}/envs/RAPIDS/bin/python {}".format(
-        conda, self.DASK_TEST_SCRIPT_FILE_NAME)
+  def verify_dask_instance(self, name, dask_runtime):
```

Comment:
I think better to remove this test, because we will run it as part of the Dask init action that should be sufficient.

Reply:
Since this init action modifies `dask-launcher.sh`, I had added this to make sure that the modifications to that file work. The current dask/rapids test doesn't technically validate Dask, only RAPIDS. I could modify this test though to also check Dask.